AUTHORS.md

```diff
@@ -0,0 +1,36 @@
+CSS
+===
+
+-   Pat Kujawa &lt;<patkujawa-wf@workiva.com>&gt; [@patkujawa-wf], Primary Author.
```

Comment:
Also, do our github accounts work as emails as well or should these be first.last@workiva.com?